src/account-info/SwitchAccountButton.js

```diff
@@ -44,8 +49,20 @@ class SwitchAccountButton extends PureComponent<Props> {
   };
 
   render() {
+    const { isOnCompatibilityScreen } = this.props;
+
+    const buttonStyle = [styles.button];
+    if (isOnCompatibilityScreen) {
+      buttonStyle.push(styles.whiteBorder);
```

Comment:
You can combine like this:
`[styles.button, isOnCompatibilityScreen && styles.whiteBorder]`

RN ignores `null`s and `false`s.